trip/models.py

```diff
@@ -1,6 +1,21 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.urls import reverse
+from ckeditor.fields import RichTextField
+
+
+class CategoryPlan(models.Model):
+    """Extended user model class that use for Category of Trip plan.
```

Comment:
I think this function doesn't extend from user model

Reply:
oh my bad. I was fix it already